tech-far-hub/src/components/site-layout.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 import { ReactNode } from "react";
-import "@trussworks/react-uswds/lib/uswds.css";
-import "@trussworks/react-uswds/lib/index.css";
+import { Link } from "gatsby";
+//import "@trussworks/react-uswds/lib/index.css";
```

Comment:
I'm placing this in a global css file and it seems to work well.

Reply:
Oh good catch --- I did not mean to just leave that sitting around.